src/PaperG/FirehoundBlob/ScenarioValidators/UnmanagedFacebookValidator.php

```diff
@@ -0,0 +1,303 @@
+<?php
+/**
+ *
+ * Author: zhiachong (zTime)
+ * Time: 7/14/16 - 3:52 PM
+ * Filename: FacebookUnmanagedValidator.php
+ */
+
+namespace PaperG\FirehoundBlob\ScenarioValidators;
+
+
+use PaperG\FirehoundBlob\CampaignData\Budget;
+use PaperG\FirehoundBlob\CampaignData\Context;
+use PaperG\FirehoundBlob\Facebook\FacebookAdSet;
+use PaperG\FirehoundBlob\Facebook\FacebookCreative;
+use PaperG\FirehoundBlob\Facebook\Targeting\FacebookAudienceTargeting;
+use PaperG\FirehoundBlob\Facebook\Targeting\FacebookDemographicTargeting;
+use PaperG\FirehoundBlob\Facebook\Targeting\FacebookGeographicTargeting;
+use PaperG\FirehoundBlob\Facebook\UnmanagedFacebookBlob;
+use PaperG\FirehoundBlob\ScenarioBlob;
+
+class UnmanagedFacebookValidator implements ScenarioValidator
+{
+    private function validateRequiredCreateFields(UnmanagedFacebookBlob $facebookBlob)
+    {
+        $validationMessage = '';
+        $validationResult  = true;
+
+        $coreFieldsValidationResults = $this->validateCoreRequiredFields($facebookBlob);
+        $validationResult &= $coreFieldsValidationResults[self::VALIDATION_RESULT];
```

Comment:
Anything && with true just has the same value, so you can use 

`validationResult = ...`